football-manager/src/lib/Pages/ContactClubAdmin/ContactPage.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+    export let id; 
+    import ContactPageNavbar from "./ContactPageNavbar.svelte";
+    import CardForContactPage from "./CardForContactPage.svelte";
+
+</script>
+
+
+<ContactPageNavbar id = {id}/>
```

Comment:
Use clubNavBar, why create another one when we already have one that is exactly the same?